carDB/src/main/java/pl/agh/edu/cardatabase/car/service/CarService.java

```diff
@@ -4,10 +4,7 @@
 import org.springframework.transaction.annotation.Transactional;
 import org.web3j.tuples.generated.Tuple2;
 import pl.agh.edu.cardatabase.blockchain.RentalBlockchainProxy;
-import pl.agh.edu.cardatabase.car.dto.CarData;
-import pl.agh.edu.cardatabase.car.dto.CarInputData;
-import pl.agh.edu.cardatabase.car.dto.CarList;
-import pl.agh.edu.cardatabase.car.dto.CarLocationUpdateInput;
+import pl.agh.edu.cardatabase.car.dto.*;
```

Comment:
The checkstyle suggests we should not use * in imports, change to imports as they were before